### ISSUE TITLE
Loading icon override plugin with multiprocess

### ISSUE THREAD

**Issue by AndOrangutan** [ibhagwan/fzf-lua#551]
### Info

- Operating System:
- Shell:
- Terminal:
- `nvim --version`: 0.8.0
- `fzf --version`: 0.34.0

- [ ] The issue is reproducible with `mini.sh`

<details>
<summary>fzf-lua configuration</summary>

```lua
require('fzf-lua').setup({
    file_icon_padding = ' ',
    winopts = {
        border = { ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '  },
        hl = {
            normal         = 'NormalFloat', -- window normal color (fg+bg)
            border         = 'NormalFloat',        -- border color (try 'FloatBorder')
            -- Only valid with the builtin previewer:
            cursor         = 'Cursor',      -- cursor highlight (grep/LSP matches)
            cursorline     = 'CursorLine',  -- cursor line
            search         = 'Search',      -- search matches (ctags)
            title       = 'FloatBorder',    -- preview border title (file/buffer)
            -- scrollbar_f = 'PmenuThumb',  -- scrollbar "full" section highlight
            -- scrollbar_e = 'PmenuSbar',   -- scrollbar "empty" section highlight
        },
    },

})

require('fzf-lua.config')._devicons_setup = '~/.config/nvim/lua/plugins/nvim-web-devicons.lua'
```
</details>

### Description 

Greetings,

When rewriting my config I wanted to integrate a larger symbol font than the "standard" [nvim-web-devicons](https://github.com/nvim-tree/nvim-web-devicons). I decided to go with [yamatsum/nvim-nonicons](https://github.com/yamatsum/nvim-nonicons), which from my understanding is a plugin that "rebinds" devicons to its replacement, nonicons. 

I can't seem to figure out how to get nonicons to work in fzf-lua. I have found issue #311 and this page in the [wiki](https://github.com/ibhagwan/fzf-lua/wiki#how-can-i-use-custom-devicons-with-multiprocess), but having 
```lua
require('fzf-lua.config')._devicons_setup = '~/.config/nvim/lua/plugins/nvim-web-devicons.lua'
```
with its contents of
```lua
require'nvim-web-devicons'.setup {
    color_icons = true;
}
```
doesn't seem to fix it, which actually makes sense to me since that other plugin does the customization (from my understanding).

**Is there an easier way of loading this other plugin to override devicons, or am I better off just defining each one in the nvim-web-devicons config?**

Also, here is how I have the fonts installed, using packer (btw your nvim config was super useful for figuring out how to use packer).
```lua 
    use { 'yamatsum/nvim-nonicons', --Icon set using nonicons for neovim plugins and settings
        requires = {'nvim-tree/nvim-web-devicons',  -- A Lua fork of vim-devicons.
            config = function () require('plugins.nvim-web-devicons') end,
        },
        --event = 'VimEnter',
    }
...
    use { 'ibhagwan/fzf-lua',   -- Improved fzf.vim written in lua
        requires = { 'yamatsum/nvim-nonicons', 'kyazdani42/nvim-web-devicons' },
        --event = 'VimEnter',
        config = function() require('plugins.fzf-lua') end,
    }

```
Also, visual confirmation the font is setup correctly and is overriding other plugins.

![image](https://user-images.githubusercontent.com/40435989/198115127-dc3ad52e-b496-485b-9b4a-bb104cb20a65.png)

Thanks in advance, and also again for sharing your awesome nvim-config :D

**Comment by ibhagwan** [comment]
Ty for the kind words @AndOrangutan :-)

Unfortunately loading `nvim-nonicons` is possible without additional hacks, multiprocess as it's name suggests runs an entirely separate neovim process with no config or plugins with the sole purpose of running a wrapper around a shell command and adding icons where files are detected using `nvim-web-devicons`, to do so fzf-lua manually loads `nvim-web-devicons` by passing the actual file path to the headless neovim instance, you can see what it does if you add debug logging by running `:FzfLua files debug=true` which should output the shell command within `:messages`, it should have a part similar to this:
```lua
'/usr/bin/nvim' -n --headless --clean --cmd ....
_G._devicons_path='.../nvim/site/pack/packer/opt/nvim-web-devicons/lua/nvim-web-devicons.lua';
```
When you add the icon customization via `_devicons_setup`, the below is also added to the command:
```lua
_G._devicons_setup='~/.config/nvim/lua/plugins/devicons.lua';
```

The two files are then loaded manually by this code:
https://github.com/ibhagwan/fzf-lua/blob/abc3aed243115dc18fc41acd49b9037655a561af/lua/fzf-lua/libuv.lua#L8-L22

The code above attempts to load each file in it's exact path (rather than the plugin runtime dir) which means it can't load any files that exist in other directories.

Unfortunately [nvim-nonicons/override.lua](https://github.com/yamatsum/nvim-nonicons/blob/main/lua/nvim-web-devicons/override.lua) contains `require` to it's mappings (in line 1) so it can't be loaded as part of the overrides (the `require` will fail), without [this line](https://github.com/yamatsum/nvim-nonicons/blob/cdb104f58c46a62ff9f484f49f8660d46db6326f/lua/nvim-web-devicons/override.lua#L1) you could theoretically use:
```lua
-- could also be inside '...site/pack/packer/opt' if plugin is optional or lazy loaded
require('fzf-lua.config')._devicons_setup = '~/.local/share/nvim/site/pack/packer/start/nvim-nonicons/lua/override.lua'
```

To summarize this long story... as it currently stands it's not possible to do what you're after without copying all the overrides to your own customization `~/.config/nvim/lua/plugins/nvim-web-devicons.lua`.
